{
  "class": "vulnerability",
  "id": "oval:com.ubuntu.impish:def:2020362220000000",
  "version": "1",
  "metadata": {
    "Title": "CVE-2020-36222 on Ubuntu 21.10 (impish) - medium.",
    "Description": "A flaw was discovered in OpenLDAP before 2.4.57 leading to an assertion failure in slapd in the saslAuthzTo validation, resulting in denial of service.\n\nUpdate Instructions:\n\nRun `sudo ua fix CVE-2020-36222` to fix the vulnerability. The problem can be corrected\nby updating your system to the following package versions:\n\nldap-utils - 2.4.57+dfsg-2ubuntu1\nlibldap-2.5-0 - 2.4.57+dfsg-2ubuntu1\nlibldap-common - 2.4.57+dfsg-2ubuntu1\nslapd - 2.4.57+dfsg-2ubuntu1\nslapd-contrib - 2.4.57+dfsg-2ubuntu1\nslapd-smbk5pwd - 2.4.57+dfsg-2ubuntu1\nNo subscription required",
    "affected": {
      "family": "unix",
      "Platform": "Ubuntu 21.10"
    },
    "reference": {
      "source": "CVE",
      "ref_id": "CVE-2020-36222",
      "ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-36222"
    },
    "advisory": {
      "Severity": "Medium",
      "Rights": "Copyright (C) 2021 Canonical Ltd.",
      "PublicDate": "2021-01-26 18:15:00 UTC",
      "Bug": [
        "https://bugs.openldap.org/show_bug.cgi?id=9406",
        "https://bugs.openldap.org/show_bug.cgi?id=9407"
      ],
      "cve": {},
      "PublicDateAtUsn": "2021-01-26 18:15:00 UTC",
      "AssignedTo": "mdeslaur",
      "DiscoveredBy": "",
      "Crd": "",
      "Ref": [
        "https://ubuntu.com/security/CVE-2020-36222",
        "https://ubuntu.com/security/notices/USN-4724-1"
      ]
    }
  },
  "notes": {
    "Note": ""
  },
  "criteria": {
    "criterions": [
      {
        "test_ref": "oval:com.ubuntu.impish:tst:2020362210000000",
        "comment": "openldap package in impish was vulnerable but has been fixed (note: '2.4.57+dfsg-2ubuntu1')."
      }
    ]
  }
}
